sdk/monitor/Azure.Monitor.Query/src/MetricsClientOptions.cs

```diff
@@ -45,5 +45,11 @@ public enum ServiceVersion
             V2024_02_01 = 1,
 #pragma warning restore CA1707 // Identifiers should not contain underscores
         }
+
+        /// <summary>
+        /// Gets or sets the audience to use for authentication with Microsoft Entra ID. The audience is not considered when using a shared key.
```

Comment:
Was the shared key sentence a carryover from another SDK? Can likely be removed since we don't support shared key credentials here.

Reply:
thanks. Will delete